content/en/metrics/guide/custom_metrics_governance.md

```diff
@@ -0,0 +1,220 @@
+
+To see all actively reporting metric names submitted by your team, go to the [Metrics Summary page][6], type in the tag key value pair (for example, `team:dev` or `service:demo`) in the **Filter by Tag Value** field.
+
+#### Identify any team's metrics that have the biggest impact on your bill
+
+All users in your organization can see OOTB realtime estimated custom metrics usage on the [Metrics Volume Management page][7]. Datadog's intelligent insights help identify which metrics to focus your cost-optimization efforts on. Use Metrics Volume Management with Metrics without Limits™, to control your indexed custom metrics usage and reduce costs without sacrificing accuracy.
+
+With Metrics Volume Management, you can identify your organization's largest metrics as well as the metric names spiking in volume (likely culprits of any unexpected overage).
+
+{{< img src="metrics/guide/volume_management_page.png" alt="Metrics Volume Management page" style="width:90%;" >}}
+
+For more information, see the [Metrics Volume Management][8] documentation.
+
+## Value-based metrics governance
+
+Effective custom metrics governance should increase monitoring efficiency. After you understand what your usage is and attribute usage to its source, take action to reduce your metrics. 
+
+In this section, you'll learn about the actions you can take to maximize the ROI and value you get from your observability spend without sacrificing the visibility your team actively relies on.
+
+### Metrics without Limits™
+
+Datadog's [Metrics without Limits™][4] is a first in industry cost management feature that decouples metric ingestion from indexing. Not all your metrics are equally valuable at every moment, and with Metrics without Limits™, you only pay for valuable metrics. 
+
+Reduce your indexed custom metrics volumes on any metric name by setting a tag configuration that you'd like to preserve for querying. It reduces your cost and preserves the mathematical accuracy of your configured metrics (all within the platform without any code-level changes).
+
+{{< img src="metrics/volume/reduce_metric_vol_cost_tags.png" alt="Example using Metrics Volume Management and Metrics without Limits™ to reduce volume by limiting the allowlist through tag configuration" style="width:80%;" >}}
+
+With Metrics without Limits™, Datadog automatically provides the following:
+- Up to date recommended tag configurations (based on our intelligent query insights) to help you maximize the ROI and value you get from your observability spend.
+
+These intelligent query insights continuously compute and analyze all users' interactions (both in-app and through the API) on any metrics submitted to us so that your recommended tag configurations are always relevant.
+- Ability to roll back changes at any time to get full visibility into all your originally submitted data.
```

Comment:
```suggestion
- Up to date recommended tag configurations (based on our intelligent query insights) to help you maximize the ROI and value you get from your observability spend.
- Intelligent query insights that continuously compute and analyze all users' interactions (both in-app and through the API) on any metrics submitted to us so that your recommended tag configurations are always relevant.
- Ability to roll back changes at any time to get full visibility into all your originally submitted data.
```